packages/plugin-block-charts/src/client/chart/antd/GroupedTable.ts

```
@@ -60,12 +60,12 @@ export class GroupedTable extends AntdChart {
     advanced?.columns?.forEach((dataValue) => {
+      if (dataValue.key === seriesField) {
+        return;
+      }
       dataSource.forEach((value) => {
-        if (dataValue.key === seriesField) {
-          return;
-        }
-        if (dataValue.calculate) {
-          if (!Number(value[dataValue.key])) {
+        if (measures?.find((item) => item.field?.join('.') === dataValue.key)) {
+          if (isNaN(Number(value[dataValue.key]))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)